### PR TITLE
S-9B

### PR DESCRIPTION
**Pull request by NudlaCZ**
Adding new scenario and proffesion.

### CODE REVIEW

File: data/json/professions.json [hunk 1]
```diff
@@ -1438,6 +1438,76 @@
...
+
+
+
+
+
+
+
+},
+
+
```
Comment (NudlaCZ) [inline data/json/professions.json]: new profession

**Comment by Fris0uman** [comment]
You need to fill the template for the Pull Request

File: data/json/professions.json [hunk 2]
```diff
@@ -1438,6 +1438,76 @@
...
   },
+{
+
+
+
+
+"type": "profession",
+    "ident": "S-9B",
+    "name": "S-9B",
+    "description": "This is profession for breach scenario.",
```
Comment (Fris0uman) [inline data/json/professions.json]: This needs an actual description of what the profession is.

File: data/json/professions.json [hunk 3]
```diff
@@ -1438,6 +1438,76 @@
...
+    "points": -3,
```
Comment (Fris0uman) [inline data/json/professions.json]: The point cost should be positive since this is a very geared up profession.

**Comment by davidpwbrown** [comment]
Did you intend to PR this to the main cleverraven repository or your own fork?

**Comment by ZhilkinSerg** [comment]
Please use https://jsonlint.com/ to find basic JSON syntax errors in JSON files you have created and/or updated.

After there are no syntax errors left, try starting game with your changes and make sure there are no loading errors and everything works correctly.

Finally use http://dev.narc.ro/cataclysm/format.html to format JSON files using our project style you have created and/or updated.

P.S.: You can make use of some docs we have here - https://github.com/CleverRaven/Cataclysm-DDA/wiki/Guide-to-adding-new-content-to-CDDA-for-first-time-contributors.

**Comment by ZhilkinSerg** [comment]
Please fix your branch and reopen PR.

**Pull request closed without merging**